.editorconfig

```diff
@@ -0,0 +1,27 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
does this make sense for python black formatting? prompt was to match prettier config.